app/models/beta/search/direct_hit.rb

```diff
@@ -0,0 +1,56 @@
+module Beta
+  module Search
+    class DirectHit
+      attr_accessor :goods_nomenclature_class,
+                    :goods_nomenclature_item_id,
+                    :producline_suffix
+
+      def id
```

Comment:
I needed an id and rather than doing a request to the db to fetch the sid it made sense just to make this what would otherwise be the unique (for today) identifier for the place in the goods nomenclature hierarchy